src/types.interfaces.ts

```diff
@@ -0,0 +1,36 @@
+export enum EventTypeEnum {
+    Unknown,
+    Navigation,
+    Location,
+    Search,
+    Transaction,
+    UserContent,
+    UserPreference,
+    Social,
+    Other,
+    Media,
+}
+
+export enum IdentityType {
```

Comment:
To avoid duplication, does it make sense to remove this from types.js? If it's easier, we can make this a sustained eng ticket and work on it separately.

Reply:
Yea I'd say this is out of scope.  It's also an issue because `IdentityType` is extended in a bizarre way where it's not just types, but methods on these types.  It needs to be completely refactored when we migrate the Identity module.  I added that to the description to discuss here: go.mparticle.com/work/SQDSDKS-2736